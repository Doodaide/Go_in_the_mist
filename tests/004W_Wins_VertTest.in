place A1
place B1
place A3
place B2
place A5
place B3
place A7
place B4
place A9
place B5