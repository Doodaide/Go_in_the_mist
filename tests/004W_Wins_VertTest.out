White wins!
A1B1A3B2A5B3A7B4A9B5
Thank you for playing!
